demonstrations/tutorial_quantum_transfer_learning.py

```diff
@@ -51,7 +51,7 @@
    new dataset :math:`D_B` and/or for a new task of interest :math:`T_B`.
 
 When dealing with hybrid systems, depending on the physical nature (classical or quantum) of the
-networks :math:`A` and :math:`B`, one can have different implementations of transfer learning as
+networks :math:`A` and :math:`B`, one can have different demonstrations of transfer learning as

```

Comment:
The find and replace was a bit too good :laughing: 
```suggestion
networks :math:`A` and :math:`B`, one can have different implementations of transfer learning as

```